frontend/e-learning-fe/src/components/userDetails.js

```diff
@@ -93,6 +106,33 @@ function UserDetails() {
               </button>
             </Link>
           </div>
+          <div class="space-y-4 box-border h-4/5 w-3/5 p-4 border-4">
+            {activities.map((activity) => (
+              <div key={activity.id}>
+                {activity.activity_description === "follow" ||
+                activity.activity_description === "unfollow" ? (
+                  ""
```

Comment:
we're not displaying when user follows/unfollows?

Reply:
Yung yung nasa wireframe sir

Reply:
lets show it as well
since in wireframe the word `activities` is used for both dashboard and profile :bow:

Reply:
Done sir